Tasks/NotationV0/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "notation",
+  "version": "0.1.0",
+  "description": "Azure Pipepine Task for setting up Notation CLI, sign and verify with Notation",
+  "main": "src/index.js",
+  "scripts": {
+    "build": "tsc -b"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "MIT",
+  "dependencies": {
+    "@types/node": "^10.17.0",
+    "@types/mocha": "^5.2.7",
+    "azure-pipelines-task-lib": "4.0.1-preview",
+    "azure-pipelines-tool-lib": "1.3.2"
```

Comment:
Could you please use tool-lib 2.x since 1.3 is officially not support node16

Reply:
If it's necessary to use 1.3.2, then please add rm in make.json, because this version uses [task-lib 3.1.10](https://www.npmjs.com/package/azure-pipelines-tool-lib/v/1.3.2?activeTab=code)

Reply:
Updated to use node 16 and tool-lib 2.x